packages/ckeditor5-table/tests/tablecolumnresize/tablecolumnresizeediting.js

```diff
@@ -482,6 +566,109 @@ describe( 'TableColumnResizeEditing', () => {
 						'</table>'
 					);
 				} );
+
+				it( 'should consume the `col[span]` attribute', () => {
```

Comment:
```suggestion
				it( 'should convert the `col[span]` attribute', () => {
```

This unit test doesn't check whether the conversion was consumed or not.